cmd/juju/model/exportbundle.go

```diff
@@ -0,0 +1,115 @@
+// Copyright 2018 Canonical Ltd.
+// Licensed under the AGPLv3, see LICENCE file for details.
+package model
+
+import (
+	"fmt"
+	"os"
+
+	"github.com/juju/cmd"
+	"github.com/juju/errors"
+	"github.com/juju/gnuflag"
+
+	"github.com/juju/juju/api/bundle"
+	"github.com/juju/juju/cmd/modelcmd"
+	"github.com/juju/juju/cmd/output"
+)
+
+// NewExportBundleCommand returns a fully constructed export bundle command.
+func NewExportBundleCommand() cmd.Command {
+	cmd := &exportBundleCommand{}
+	cmd.newAPIFunc = func() (ExportBundleAPI, error) {
+		return cmd.getAPI()
+	}
+	return modelcmd.Wrap(cmd)
+}
+
+type exportBundleCommand struct {
+	modelcmd.ModelCommandBase
+	out        cmd.Output
+	newAPIFunc func() (ExportBundleAPI, error)
+	// name of the charm bundle file.
```

Comment:
Either remove or re-phrase. At this stage, with information this comment contains it is not helpful :)